components/org.wso2.micro.integrator.initializer/src/main/java/org/wso2/micro/integrator/initializer/handler/transaction/store/connector/RDBMSConnector.java

```diff
@@ -112,11 +112,12 @@ public synchronized void addTransaction() throws TransactionCounterException {
      * @throws TransactionCounterException Error occurred while fetching data from the database.
      */
     public long getTransactionCountOfMonth(int year, int monthNumber) throws TransactionCounterException {
-
-        String dateString = year + "-" + monthNumber + "-01";
+        String monthNumberStr = Integer.toString(monthNumber);
+        monthNumberStr = ((monthNumberStr).length() == 1) ? "0" + monthNumberStr : monthNumberStr;
```

Comment:
shall we remove all the enclosing brackets .
or at least the one around (monthNumberStr)

Reply:
Fixed